src/pages/index.tsx

```diff
@@ -30,15 +34,28 @@ type Props = {
   lastFetched: string;
 };
 
+type RecordFilter = (record: ApiRecord) => boolean;
+
 const IndexPage = ({ records, lastFetched }: Props) => {
   const [moduleFilter, setModuleFilter] = useState<Module | null>(null);
-  const filteredRecords = useMemo(
-    () =>
-      moduleFilter
-        ? records.filter((record) => record.module === moduleFilter)
-        : records,
-    [records, moduleFilter]
-  );
+  const [minVerFilter, setMinVerFilter] = useState<string | null>(null);
+  const filteredRecords = useMemo(() => {
+    if (!moduleFilter && !minVerFilter) {
+      return records;
+    }
+    const minVer = minVerFilter?.startsWith("v")
+      ? minVerFilter?.slice(1)
+      : minVerFilter;
+    const f: RecordFilter = moduleFilter
+      ? (record) => record.module === moduleFilter
+      : (_) => true;
+    const g: RecordFilter = minVer
+      ? (record) => {
+          return semverCompare(record.supported.slice(1), minVer) >= 0;
```

Comment:
Note to reviewer) `.slice(1)` is to drop the leading `v` in version string.
This is required because `semver-compare` does not accept `v1.2.3`